_extras/design.md

```diff
@@ -4,35 +4,160 @@ title: Lesson Design
 
-## Required Pre-Knowledge
+#### Barbara from Material Science
+
+Barbara just started her PostDoc in Material Science. Her new group has a large amount of scanning electron miscroscope images stored which exhibit several metals when exposed to a plasma. The team also made the effort to highlight solid deposits in these images and thus obtained 20,000 images with such annotations. Barbara performed some image analysis before and hence has the feeling that Deep Learning may help her in this task. She saw her labmates use ML algorithms for this and is motivated to finally understand these approaches.
+
+#### Dan from Life Sciences
+
+Dan produced a large population of bacteria that were subject to genetic alterations resulting in 10 different phenotypes. The latter can be identified by different colors, shapes and movement speed under a fluorescence microscope. Dan has not a lot of experience with image processing techniques to segment these different objects, but used GUI based tools like [fiji](https://fiji.sc) and others. He has recorded 50-60 movies of 30 minutes each. 10 of these movies have been produced with one type of phenotype only. Dan doesn't consider himself a strong coder, but would need to identify bacteria of the phenotypes in the dataset. He is interested to learn if Deep Learning can help.
+
+#### Eric from Pediatrics Science
+
+Eric ran a large array of clinical trials in his hospital to improve children pharmaceutics for treating a common (non-lethal) virus. He obtained a table that lists the progression of the treatment for each patient, the dose of the drug given, whether the patient was in the placebo group or not, etc. As the table has more than 100 000 rows, Eric is certain that he can use ML to cluster the rows in one column where the data taking was inconsistent. Eric has touched coding here and there where necessary, but never saw it necessary to learn coding. His cheatsheet is his core wisdom with code. So his supervisor invited him to take a course on ML as "this is the tech of these days!" as his boss said.
+
+### Notes
+
+- Probably have overhyped expectations of deep learning. 
+- They don’t know if it’s the right tool for their situations. 
+- They have no idea what it takes to actually do deep learning.
+- Want to quickly have some useful skills for their own data.
+
+#### Required Pre-Knowledge
 
-- **Basic Machine Learning Knowledge** – Data Cleaning, Train & Test Split, Overfitting & Underfitting, Metrics (Accuracy, Recall, etc.),
 - **Python** – Previous Experience programming in python is required (Refer to Python Data Carpentry Lesson )
 - **Pandas** – Knowledge of the Pandas python package
+- **Basic Machine Learning Knowledge** – Data Cleaning, Train & Test Split, Overfitting & Underfitting, Metrics (Accuracy, Recall, etc.),
```

Comment:
```suggestion
- **Python** – Previous programming experience in Python is required (Refer to Python Data Carpentry Lesson)
- **Pandas** – Knowledge of the Pandas Python package
- **Basic Machine Learning Knowledge** – Data cleaning, train & test split, overfitting & underfitting, metrics (accuracy, recall, etc.),
```

Reply:
only nitty-gritty stuff, trying to make it more consistent (feel free to change/ignore).